packages/asset/contracts/Asset.sol

```diff
@@ -78,6 +80,7 @@ contract Asset is
         __ERC1155Supply_init();
         __ERC2771Handler_init(forwarder);
         __ERC1155Burnable_init();
+        __Ownable_init();
```

Comment:
Using both AccessControlUpgradeable and OwnableUpgradeable is overkill, is enough with access control.
If you decide to keep ownable then it is a bad idea to set the owner to msg.sender (deployer), is better to pass an admin address to the initialize function and set that address as owner (with this old version you must call transferOwnership).